debezium-assembly-descriptors/src/main/resources/assemblies/connector-distribution-no-drivers.xml

```diff
@@ -19,8 +19,6 @@
               <exclude>com.fasterxml.jackson.core:jackson-core:*</exclude>
 
               <!-- Exclude connection drivers -->
-              <exclude>mysql:mysql-connector-java:*</exclude>
```

Comment:
I don't quite get this, won't it re-addd the upstream drivers?

Reply:
@gunnarmorling No, upstream build will use upstream drivers, prodcut build will use product drivers - the GAV is automatically modified by build system.

Reply:
Ok, got it.